llvm/docs/ReleaseNotes.rst

```diff
@@ -153,6 +153,7 @@ Changes to the RISC-V Backend
   "SiFive Custom Instruction Extension" as SiFive defines it. The LLVM project
   needs to work with SiFive to define and document real extension names for
   individual CSRs and instructions.
+* ``-mcpu=sifive-p450`` was added.
```

Comment:
I think this patch is not for `-mcpu`, we have already added it before. This patch adds the scheduling model, and it doesn't affect any noticeable options.

Reply:
It still needs to be in the release notes and should have been flagged in the previous review. I can commit this directly if you prefer?

Reply:
> I can commit this directly if you prefer?

Yes, that's OK.